RICH_Data_Overview.ipynb

```diff
@@ -1,575 +1,587 @@
QgXm72BtnNBSP736zMeWjkvsp7jdVNGAuuWkR0/InX4krZdyBjW7W2Hy6NV7ooZTm+i+Hf33zrjjP90W3gV4Isv9nEP1dR97Wt9fPxLlEjBDdC4fhMVF8bLL/uY2xZcU4eWNN5LxxvHBYSjosyvUG0H/l7L1V/KFW2cnNjGo/7Z7EX+HuSlPOl57o3aWNTSw9MUc8ZJrs5BLhvyqOg2t1ty7Ss+dq6NFb+O0/i9PuMMH//P/+nCH+722Yyf/azf/b3vHX2WQCd9X4tI3mh/v802s78CADM7CMAqAA+nH9IUWwEcF8WzUMweFxERaZ19aYv1/lc9ajuIiIgMp0Jth9FexTsA/PNQw+DXAHwnhPCF5l3WiO4DMM/MegE8DmAFeMiTrFu3mUYWPkh78CyxGs5I4FGGcbupv51XbaCln7B9u49ppQb3oeCe4YwJmWWmyiwdlasSnctQyO2f6sUqu8pD2YrHKbxvmRUjhosZvwex3HXztfDoS+o1L1wXVxTnzyFnzjQQj8zn5jUWl6gcfUZCo2smlKH5mu3n5+PauqaeTDUURtK1bQeW+zdstnr/bVoQCPPn+5j/5PIiTmz3r/r6kSfOojzCeHR98WK/LZeRwBdHNXbG8fH4cx7F3O6YRN8lh3AbiZc7piWop/IS1VG7aGqujXTwwentbNYsH9PzHBe9aUetWuW25b77+fub4zIZCjM4syLXzsxlYvDy1kceWbu90K/4VDjXSv99/nWfUIj+fs5mFBGnQm2H5LMws1Oj8IsA/gGD6y7+u5mdGkK4v5kXx0IIu83sMgB3YHDpp+tCCA+08hpERERkZGo7iIiIjB25bpHPUTyAwSWXPofB+Ye/3oyLSgkh3AbgttHuv2TJXPT1pUYx17j4wgtfvf82dbbz1HI8+qQfJ57Nwwzcu849xTwSHMfck8/H5t55OlduFQDuIT8ksa1s73s9WQP11mvgeZSpc/NrwucqO98zd63x/lynIjfXNJc1ws8lNeDF5z4kVcsDwM9/fpSLJ05c5uJdu7a5OM5CKLMqw/BupGOnV26It7czK0AZCWNMhUYZGqTr2w6sTNbRwIBf5WHTJl8U4b3v9TWRVq701e8//Wkk402z/DfdtGm1eOFC//d6OjdcuGbODlpzgtMrODXzoIN8HGcVUC0nPPVU+tx8rIce8jG3g+Jr4ZF2PhbVgii0tzgjgUffuf3205/uvzmVsjz20mvG38+8qgevlpBbkcJ9vy9b5jfy+8sN1xkzfMx/p/j9jV8Xfo3PPNOFN9zq15y46qrV8Pz7qe9FETJWVnkIIfzaSNvM7KLGX07zmf2M7ul1UfwdknuPZ8+gn2Zb6WuE8xpzeY7xl2fu5PzYF15wYe6HOR89fiZl0umHk1u+KX58brnEXHHC3BSJVJx7jert3Cj7ujVSaspD4ZPFDZIGJpkAACAASURBVApqWHFG7K5d3F0x00W+cOLptI3//eX4rORGTiXQtITu1lHvX4d2KJjZhQB+G4NZ018JIXy3FeetYtsh//m6ZsQtAwPvpnsWueimm3zboLfXt0tuuSV95rPOqt2mGQtYvNh/C86de7yLn4SPj7+Q/uAznkIRf39wijx1COxd+BoXj7v3h35/et6Fc8VfRvxE2aJF6e38xcY/nrm9Nm9e7TYV3J5GHQq8VHPZosvcbnHdR/SjPvs8eIojo2kmcafPK/P9+8V1xm+/nQ828r+BQR/ObG+ejvq+ENmncW2Htq/yULaofuzzDbsKERGRbtaiwkpmdp2ZPW1mG+n+c8zsITPbbGYfAYAQwi0hhHcDuBjA7zf6KR8gtR1ERESAMVeUcTjDLcPUBfwSd/Pm+V7KOEOPM8U4W6/QQ85ZA7xcU66nOO5Z5p55ljlW2SJ+qZF7lls+kbfnzh0r23OfS/3na4tjPhdPBcidi+WmTMTnrncJytzj49ehsJwWT71hlAK7e/fxtMO9FP+AYt4/9k/JU9ezLOTg42ujELl9NULR3cbw+3c9gC8DuGHfHUPLMX4FwHIMrmhwn5mtDiHsW2/wY0PbO0FXth1yI5z1FIgFfAZDf/9siv3evb2+mGGc5c7FpP+J/uS+7W0+5nbO7rk+Ib/YzPEFIuMEBZ6JuZTGyzbTbIojTni9i3mWwgS6tvj4sxf6zLpXZvnvHZ658ZpZfjHHH27y6fo842H3bj915MQ5UXDaaX7ne/331BQqjJhrW/AUh0KRxjg74uyz3bZHjvGv4fGzfMuDp+ZyAgPNcMTD0WyOX6UL+frXfZxLEikurdo+Y/j7QqQl6ulQKKzhLCIiMua0cB5kCOFuM5tDj3sdgM0hhEcGL8e+AeACM/sJgCswuLpCSwshJqjtICIi0qHTJQ9EbpWHDRj+y98AHDPM/R1n3bpf0NJzftnIhx/2WQVnnVXrkuWe+wFfRwk4mYr6cDEcLgLEGQu8PX48ZzvwqHJunamS4h5z/lDwyDuP1HNvem4kP+4vz52L8XY+d24OT+r4ZYssslymRnw8qlqQnWOZOzZvj7MSCotr5Zbbog9+cRXJHoo5I+GR/bdC8OtI5ZeJ5AyF05HC+6umgrRF++dBHgsgroi3FcBpAN4H4GwAh5vZ3BDCVQ24xqwqtB1YfX8PLqd427B71TxKMS9v7f8Gx6UKbr3V70kD2li/Pn1mHnXesMHHM33JHLdKJTdL1qzxMTeRTqc/73yuww4b6SqB9ZQ595N/8duXL/fx6nt8RgKvnskZqBy/OKuWsXAIXxg9ken8xKnAI3+lcnugIEqfeGWpz0i49hN+18MP9xkJxx3nt19/vY/POcfHG6OJVPw0br7ZxwMD+s4UqctY6VAAcF5mu4iIiLTXcNMIQgjhSwC+1OqLgdoOIiIiY0ZulYcRS7Kb2Q8AvKHhV9RgS5ZMR19fvIzdGrd94kQ/WnvkkbXbvJoSz7F7ZbKfaziJH5Bbnoe761O9VDQnrzDKTKs88JE4n4FHwOMRbd43N3KfGy1PrZaQW2Ky7
```

Comment:
Same comment as in score.py: you need to concatenate X and Y.

Reply:
Yeah, but this is a sample notebook and for the sake of simplicity i guess we should left it as it was (with y only)

Reply:
Since it introduces the students to the task, I'd prefer it to have the same metric as the competition.